src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+
+  @for $index from 1 through 31 {
+    &__day--#{$index} {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      border: 1px solid black;
+      box-sizing: border-box;
+      width: $colWidth;
+      height: 100px;
+
+      background-color: #eee;
+
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      text-align: center;
+    }
+    &__day--#{$index}::before {
+      content: "#{$index}";
+    }
+  }
+
+  @each $day, $num in $days {
+    &--start-day--#{$day} > &__day--1 {
+      margin-left: getMargin($colWidth, $calendarPadding, $gap, $num);
+    }
+  }
+
+  & > div {
+    transition: background-color 0.5s, transform 0.5s;
```

Comment:
ı've used variables for tranition time. Maybe you want to consider using var

Reply:
Also can be added under `calendar__day` class.